Makefile

```
@@ -274,4 +274,4 @@ fuzz-vara:
 
 .PHONY: kill
 kill:
-	@ pgrep -f "gear-node" | xargs kill -9
+	@ pgrep -f "release/gear" | xargs kill -9
```

```suggestion
	@ pgrep -f "gear" | xargs kill -9
```

That's executable binary name (process name)

---

As it is a regular expression, just `gear` also kills `gear.sh` which yields an error.

---

Replaced it with `pkill -f 'gear |gear$' -9`, seems to work now.